dlt/common/storages/load_package.py

```diff
@@ -380,6 +386,30 @@ def save_schema_updates(self, load_id: str, schema_update: TSchemaTables) -> Non
         ) as f:
             json.dump(schema_update, f)
 
+    #
+    # Loadpackage state
+    #
+    def get_load_package_state(self, load_id: str) -> DictStrAny:
+        package_path = self.get_package_path(load_id)
+        if not self.storage.has_folder(package_path):
+            raise LoadPackageNotFound(load_id)
+        try:
+            state = self.storage.load(
+                os.path.join(package_path, PackageStorage.LOAD_PACKAGE_STATE_FILE_NAME)
+            )
+            return cast(DictStrAny, json.loads(state))
+        except FileNotFoundError:
+            return {}
```

Comment:
Is logging details is useful for debugging?

Reply:
no, this just means that there is no statefile yet